src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/upgrade/upgrade.component.html

```diff
@@ -21,30 +27,44 @@
              [ngStyle]="healthData.health.status | healthColor"
              [ngbPopover]="healthChecks"
              popoverClass="info-card-popover-cluster-status">
-             {{ healthData.health.status | healthLabel | uppercase }}
+            {{ healthData.health.status | healthLabel | uppercase }}
           <i *ngIf="healthData.health?.status !== 'HEALTH_OK'"
              class="fa fa-exclamation-triangle"></i>
         </div>
       </div>
-      <div class="col-lg-3 h-50 d-flex flex-column border justify-content-center align-items-center">
-        <span class="bold">Upgrade Status</span>
-        <ng-container *ngIf="upgradeInfo.versions.length > 0; else noUpgradesAvailable">
-          <span class="mt-2"
-                i18n>
-          <i [ngClass]="[icons.up]"
-             class="text-info"></i>
-            Upgrade available</span>
-          <div i18n-ngbTooltip
-               [ngbTooltip]="(healthData.mgr_map | mgrSummary).total <= 1 ? 'To upgrade, you need minimum 2 mgr daemons.' : ''">
-            <button class="btn btn-accent mt-2"
-                    id="upgrade"
-                    aria-label="Upgrade now"
-                    [disabled]="(healthData.mgr_map | mgrSummary).total <= 1"
-                    i18n>Upgrade now</button>
-          </div>
+    </cd-card>
+
+    <cd-card class="col-sm-3 px-3 d-flex"
+             aria-label="Upgrade Status"
+             id="upgradeStatus"
+             *ngIf="{upgradeInfo: upgradeInfo$ | async, error: upgradeInfoError$ } as upgrade">
+      <div class="d-flex flex-column justify-content-center align-items-center">
+        <span class="bold">Status</span>
+        <ng-container *ngIf="!upgrade.upgradeInfoError && upgrade.upgradeInfo as upgradeInfo; else checkingForUpgradeStatus">
+          <ng-container *ngIf="upgradeInfo.versions.length > 0; else noUpgradesAvailable">
+            <span class="mt-2"
+                  i18n>
+            <i [ngClass]="[icons.up]"
+               class="text-info"></i>
+               New version {{ upgradeInfo.versions[0] }} available</span>
+            <div i18n-ngbTooltip
+                 [ngbTooltip]="(healthData.mgr_map | mgrSummary).total <= 1 ? 'To upgrade, you need minimum 2 mgr daemons.' : ''">
+              <button class="btn btn-accent mt-2"
+                      id="upgrade"
+                      aria-label="Upgrade now"
+                      [disabled]="(healthData.mgr_map | mgrSummary).total <= 1"
```

Comment:
Shouldn't this be disabled if the cluster health is ERR? In your screenrecording, it's in Cluster Health ERR, but allowing to upgrade. That's what I meant, that I'd remove the cluster health and mgr count if the Upgrade button is responsive to those conditions.

Reply:
Nope. Even if the cluster is unhealthy the upgrade will proceed. The health will flicker while the upgarde is in progress as well. When we designed this, we thought it would be nice to show the cluster health to the user upfront before proceeding with the upgrade.